README.md

```diff
@@ -1400,7 +1400,7 @@ Okay, now it's deleted :confused:
 + `del x` doesn’t directly call `x.__del__()`.
 + Whenever `del x` is encountered, Python decrements the reference count for `x` by one, and `x.__del__()` when x’s reference count reaches zero.
 + In the second output snippet, `y.__del__()` was not called because the previous statement (`>>> y`) in the interactive interpreter created another reference to the same object, thus preventing the reference count to reach zero when `del y` was encountered.
-+ Calling `globals` caused the existing reference to be destroyed and hence we can see "Deleted!" being printed (finally!).
++ Calling `globals` caused the existing reference to be destroyed and hence we can see “Deleted!” being printed (finally!).
```

Comment:
I see a lot of quotes replaced (" with “) in the PR. Is there any difference in the meanings of both the characters. The one you replace it with isn't a character typed by standard keyboard.

Reply:
They have the same meaning. These are the modern, better looking, more readable, English opening `“` and closing `”` quotes. There are easily available – though not widely known – on modern keyboard layouts.

If you don’t like it though, I can discard that change from my PR.

Reply:
> They have the same meaning. These are the modern, better looking, more readable, English opening “ and closing ” quotes. There are easily available – though not widely known – on modern keyboard layouts.

Interesting, I didn't knew that :)

> If you don’t like it though, I can discard that change from my PR.

The thing is, they are not ASCII. And given the fact anyone can contribute to the project, it'd be quite tedious to maintain consistency for quotes in the future examples. I hope you understand...

Reply:
No problem, I totally understand. :)